src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -298,5 +312,7 @@
              :account-props       (if (= transaction-type :tx/bridge)
                                     from-account-props
                                     user-props)
+             :recipient           recipient
              :network-values      to-values-by-chain
+             :account-to?         true
```

Comment:
what is this prop for?

Reply:
There is two usages of `user-summary` component in this file which one is for the `account-from` and the other is for `account-to`. If the prop is true, then we know which one we're handling

Reply:
ok cool, just wondering if we are using `receiver` in some cases and `account-to` in others. Probably it's best to use one and just be consistent with it.